docusaurus.config.js

```diff
@@ -42,6 +42,12 @@ module.exports = {
       },
     ],
     "./static/plugins/bpmn-js",
+    [
+      "docusaurus-pushfeedback",
+      {
+        project: "m6exeps3n1",
```

Comment:
@christinaausley Consider adding the following options to avoid the overlap with the ASK AI button:

```suggestion
        project: "m6exeps3n1",
        buttonStyle: "dark",
        buttonPosition: "center-right",
 ```
 
 Result:
 
![image](https://github.com/camunda/camunda-docs/assets/9107969/a19df85b-7048-4ba8-93bd-b04383676c74)
 
 Optionally, you could choose to open the feedback widget as a sidebar:
 
 ```suggestion
        project: "m6exeps3n1",
        buttonStyle: "dark",
        buttonPosition: "center-right",
        modalPosition: "sidebar-right",
 ```

Result:

![image](https://github.com/camunda/camunda-docs/assets/9107969/283edf34-f076-4dd0-8843-89e8881c1a2c)

Hope it helps!

Reply:
Restarting with npm worked! Thank you so much!